images/techdocs-cli/context/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM docker.io/library/python:3.10.7@sha256:e9c35537103a2801a30b15a77d4a56b35532c964489b125ec1ff24f3d5b53409 as base
+
+FROM base as python
+
+RUN mkdir /srv/workspace
+WORKDIR /srv/workspace
+
+FROM python as pip-tools
+
+COPY requirements-pip-tools.txt requirements-pip-tools.txt
+RUN pip install \
+    --no-cache-dir \
+    --require-hashes \
+    --verbose \
+    --use-pep517 \
+    --requirement requirements-pip-tools.txt && \
+    rm -rf requirements-pip-tools.txt
+
+FROM python as techdocs-cli
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y apt-transport-https=2.2.4 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash
+
+RUN apt-get install --no-install-recommends -y \
+    nodejs=18.10.0-deb-1nodesource1 \
+    openjdk-17-jdk=17.0.4+8-1~deb11u1
+
+# Download plantuml file, Validate checksum & Move plantuml file
+RUN curl -o plantuml.jar -L http://sourceforge.net/projects/plantuml/files/plantuml.1.2022.4.jar/download && echo "246d1ed561ebbcac14b2798b45712a9d018024c0  plantuml.jar" | sha1sum -c - && mv plantuml.jar /opt/plantuml.jar
```

Comment:
can we not use the plantuml docker image? https://hub.docker.com/r/plantuml/plantuml and copy it from there, then we can use dependabot for updates on this image

Reply:
fixed